ui/src/shared/utils/useVisDomainSettings.ts

```diff
@@ -54,3 +54,42 @@ export const useVisDomainSettings = (
 
   return [domain, setDomain, resetDomain]
 }
+
+export const getRemainingRange = (
+  data: NumericColumnData = [],
+  timeRange: TimeRange | null,
+  storedDomain: number[]
+) => {
+  const range = extent((data as number[]) || [])
```

Comment:
It is necessary to set a default for the range since the output of the `extent` function can be null, in which case we would want to return an empty array.

Reply:
so does that mean it should be 

```ts
  const range = extent(data as number[]) || []
```

Reply:
Aw, I think I see what you were getting at. Ya, that's a bit awkward. Honestly I don't think I took an introspective look at the existing code to analyze why things were set they were there. You're totally right